### PR TITLE
Updated Notebook for INC 2.0

### PR DESCRIPTION
**Pull request by asirvaiy**
## Type of Change

  API changed 

## Description

Updated for INC 2.0 

## Expected Behavior & Potential Risk

It will enable the notebook to run on INC 2.0

## How has this PR been tested?

Env details are there to create an env

## Dependency Change?

TensorFlow is latest 2.11 now.

### CODE REVIEW

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """
 Environment Setting
 
-Enable Intel Optimized TensorFlow 2.6.0 and newer by setting environment variable TF_ENABLE_ONEDNN_OPTS=1
-That will accelerate training and inference, and  it's mandatory requirement of running Intel® Neural Compressor quantize Fp32 model or deploying the quantized model.
+Enable IntelÂ® oneDNN Optimizations in TensorFlow 2.6.0 and newer by setting environment variable TF_ENABLE_ONEDNN_OPTS=1. In TensorFlow 2.9 and newer, oneDNN is enabled by default.
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py:4]: -> Intel® oneDNN Optimizations

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py [hunk 2]
```diff
@@ -27,13 +31,14 @@
 
 def preprocess(image, label):
     image = tf.cast(image, tf.float32)/255.0
-    return tf.image.resize(image, [w, h]), tf.one_hot(label, class_num)
+    return  tf.image.resize(image, [w, h]), np.array(tf.one_hot(label, class_num))
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py:34]: np.array() return numpy array.
tf.one_hot return tf.tensor.
preprocess() should return (tf.tensor, tf.tensor).
please rm np.array() if code is passed.

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb [hunk 3]
```diff
@@ -364,22 +346,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from neural_compressor.experimental import Quantization, common\n",
-    "\n",
-    "def preprocess_1(image, label):\n",
+    "def preprocess(image, label):\n",
     "    image = tf.cast(image, tf.float32)/255.0\n",
-    "    return tf.image.resize(image, [w, h]), label\n",
+    "    return  tf.image.resize(image, [w, h]), np.array(tf.one_hot(label, class_num))    \n",
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb:351]: rm np.array if code is passed.

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb [hunk 4]
```diff
@@ -364,22 +346,28 @@
...
+    "                                       as_supervised = True, \n",
+    "                                       split = ['train', 'test'])\n",
+    "                                       \n",
+    "    return raw_datasets, raw_info\n",
+    "    \n",
     "class Dataset(object):\n",
     "    def __init__(self):\n",
     "        datasets , info = load_raw_dataset()        \n",
-    "        self.train_dataset = [preprocess_1(v, l) for v,l in datasets[0]]\n",
+    "        self.test_dataset = [preprocess(v, l) for v,l in datasets[-1]]\n",
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb:364]: original code use train_dataset. It will help get good accuracy.
please use train_dataset.

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb [hunk 5]
```diff
@@ -364,22 +346,28 @@
...
     "    \n",
     "    def __getitem__(self, index):\n",
-    "        return self.train_dataset[index]\n",
+    "        return self.test_dataset[index]\n",
     "\n",
     "    def __len__(self):\n",
-    "        return len(list(self.train_dataset))"
+    "        return len(list(self.test_dataset))\n"
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb:370]: please use train_dataset if possible.

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py (hunk 2 above)
Comment (asirvaiy) [reply examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py:34]: WIth INC 2 TF dataloader, if we use tf.tensor for labels, it causes that labels and preds type not matching issue. 
we can directly use labels. Let me change it.

Comment (NeoZhangJianyu) [reply examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py]: OK! ignore my comment

File: examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb (hunk 4 above)
Comment (asirvaiy) [reply examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb:359]: Updated again now.

**Comment by chensuyue** [comment]
please sign up by click here: 
image

**Comment by chensuyue** [comment]
@NeoZhangJianyu please approve if no comments.

**Review by NeoZhangJianyu — APPROVED** [review]
Approve
